examples/step-93/step-93.cc

```diff
@@ -0,0 +1,840 @@
+  {
+    // First, we do a standard loop setup for constructing the system matrix.
+    hp::FEValues<dim> hp_fe_values(fe_collection,
+                                   quadrature_collection,
+                                   update_values | update_gradients |
+                                     update_quadrature_points |
+                                     update_JxW_values);
+
+    FullMatrix<double> cell_matrix;
+    Vector<double>     cell_rhs;
+
+    std::vector<types::global_dof_index> local_dof_indices;
+
+    const FEValuesExtractors::Scalar u(0);
+    const FEValuesExtractors::Scalar lambda(1);
+    const FEValuesExtractors::Scalar c(2);
+
+    for (const auto &cell : dof_handler.active_cell_iterators())
+      {
+        const unsigned int dofs_per_cell = cell->get_fe().n_dofs_per_cell();
+
+        cell_matrix.reinit(dofs_per_cell, dofs_per_cell);
+        cell_rhs.reinit(dofs_per_cell);
+        hp_fe_values.reinit(cell);
+
+        cell_matrix = 0;
+        cell_rhs    = 0;
+
+        const FEValues<dim> &fe_values = hp_fe_values.get_present_fe_values();
+
+        local_dof_indices.resize(fe_values.dofs_per_cell);
+
+        cell->get_dof_indices(local_dof_indices);
+
+
+        // In the loop over quadrature points, we start by building
+        // all of the usual terms that are bilinear in shape functions
+        // corresponding to the $u$ and $\lambda$ variables:
+        for (const unsigned int q_index : fe_values.quadrature_point_indices())
+          {
+            const double JxW = fe_values.JxW(q_index);
+            for (const unsigned int i : fe_values.dof_indices())
+              {
+                const double phi_i_u = fe_values[u].value(i, q_index),
+                             phi_i_l = fe_values[lambda].value(i, q_index);
+
+                const Tensor<1, dim> grad_i_u =
+                                       fe_values[u].gradient(i, q_index),
+                                     grad_i_l =
+                                       fe_values[lambda].gradient(i, q_index);
+
+                for (const unsigned int j : fe_values.dof_indices())
+                  {
+                    const double phi_j_u = fe_values[u].value(j, q_index);
+
+                    const Tensor<1, dim> grad_j_u =
+                                           fe_values[u].gradient(j, q_index),
+                                         grad_j_l =
+                                           fe_values[lambda].gradient(j,
+                                                                      q_index);
+
+                    cell_matrix(i, j) += phi_i_u * phi_j_u * JxW;
+                    cell_matrix(i, j) += -grad_i_u * grad_j_l * JxW;
+                    cell_matrix(i, j) += -grad_i_l * grad_j_u * JxW;
+                  }
+
+
+                // Here, we actually create a quadrature point object (rather
+                // than an index), and use this to construct the RHS of the
+                // linear system.
```

Comment:
```suggestion
```
I don't think this is worth noting since this is exactly how we handle forcing functions in all Laplace solver examples.